shell/platform/android/io/flutter/embedding/engine/systemchannels/TextInputChannel.java

```diff
@@ -360,8 +368,9 @@ public void setTextInputMethodHandler(@Nullable TextInputMethodHandler textInput
      * different client is set.
      *
      * @param id the ID of the platform view to be set as a text input client.
+     * @param usesVirtualDisplay Whether the platform view uses a virtual display.
```

Comment:
May be helpful to mention the alternative to VD here

Reply:
Done